src/NovelRT/Graphics/RenderingService.cpp

```diff
@@ -20,10 +15,9 @@ namespace NovelRT::Graphics {
 
     if (completeLaunch) {
       _camera = Camera::createDefaultOrthographicProjection(windowSize);
-      _openGLContext = SDL_GL_CreateContext(_runner->getWindowingService()->getWindow());
-      SDL_GL_MakeCurrent(_runner->getWindowingService()->getWindow(), _openGLContext);
+      glfwMakeContextCurrent(_runner->getWindowingService()->getWindow());
 
-      if (!gladLoadGL()) {
+      if (!gladLoadGLLoader((GLADloadproc) glfwGetProcAddress)) {
```

Comment:
C-style cast should be replaced with the appropriate C++-style cast, or the compiler is going to guess/assume which is dangerous.